packages/service/lib/generator/service-generator.js

```diff
@@ -21,6 +21,32 @@ class ServiceGenerator extends BaseGenerator {
     }
   }
 
+  getConfigFields () {
```

Comment:
These should not be set in the case of a runtime app.

Reply:
Fixed in [23d8c33](https://github.com/platformatic/platformatic/pull/1846/commits/23d8c33a5158df58bed1b3dd80baa8b4b14273f9)